modules/oc-mirror-command-reference.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: REFERENCE
+[id="oc-mirror-command-reference_{context}"]
+= Command reference for oc-mirror
+
+The following tables describe the `oc mirror` subcommands and flags:
+
+.oc mirror subcommands
+[cols="1,2",options="header"]
+|===
+|Subcommand
+|Description
+
+|`completion`
+|Generate the autocompletion script for the specified shell.
+
+|`describe`
+|Output the contents of an image set.
+
+|`help`
+|Show help about any subcommand.
+
+|`list`
+|List available platform and Operator content and their version.
+
+|`version`
+|Output the oc-mirror version.
+
+|===
+
+.oc mirror flags
+[cols="1,2",options="header"]
+|===
+|Flag
+|Description
+
+|`-c`, `--config` `<string>`
+|Specify the path to an image set configuration file.
+
+|`--continue-on-error`
+|If any non image-pull related error occurs, continue and attempt to mirror as much as possible.
+
+|`--dest-skip-tls`
+|Disable TLS validation for the target registry.
+
+|`--dest-use-http`
+|Use plain HTTP for the target registry.
+
+|`--dry-run`
+|Print actions without mirroring images.
+
+|`--from <string>`
+|Specify the path to an image set archive that was generated by an execution of oc-mirror to load into a target registry.
+
+|`-h`, `--help`
+|Show the help.
+
+|`--log-level <string>`
+|Specify the number for the log level verbosity. Valid values are `0` - `9`. The default is `0`.
+
+|`--manifests-only`
+|Generate manifests for `ImageContentSourcePolicy` objects to configure a cluster to use the mirror registry, but do not actually mirror any images.
+
+|`--skip-cleanup`
+|Skip removal of artifact directories.
+
+|`--skip-image-pin`
+|Do not replace image tags with digest pins in Operator catalogs.
+
+|`--skip-missing`
+|If an image is not found, skip it instead of reporting an error and aborting execution. Does not apply to custom images explicitly specified in the image set configuration.
+
+|`--skip-verification`
+|Skip digest verification.
+
+|`--source-skip-tls`
+|Disable TLS validation for the source registry.
+
+|`--source-use-http`
+|Use plain HTTP for the source registry.
+
```

Comment:
Not sure if this is intentional, but the following items are missing from 4.10 when compared to 4.11:

![Screenshot from 2023-06-07 11-58-31](https://github.com/openshift/openshift-docs/assets/57954076/056c899f-6d5c-4eec-8d6e-90f185927477)

Reply:
The 4.10 docs do link to https://docs.openshift.com/container-platform/4.10/updating/updating-restricted-network-cluster/index.html